pkg/model/issuerdiscovery.go

```diff
@@ -80,7 +80,7 @@ func (b *IssuerDiscoveryModelBuilder) Build(c *fi.ModelBuilderContext) error {
 	keysFile := &fitasks.ManagedFile{
 		Contents:  keys,
 		Lifecycle: b.Lifecycle,
-		Location:  fi.String("/openid/v1/jwks"),
+		Location:  fi.String("oidc/openid/v1/jwks"),
```

Comment:
@h3poteto please also change the `.well-known` path below on line 93. Also, please remove the appending of "oidc" around line 125 in `pkg/model/iam/subject.go`